deepchecks/nlp/context.py

```diff
@@ -39,13 +39,14 @@
     'TTokenPred'
 ]
 
-from deepchecks.utils.validation import is_sequence_not_str
 
 TClassPred = t.Union[t.Sequence[t.Union[str, int]], t.Sequence[t.Sequence[t.Union[str, int]]]]
 TClassProba = t.Sequence[t.Sequence[float]]
+# TODO: is it correct, why tuple?
 TTokenPred = t.Sequence[t.Sequence[t.Tuple[str, int, int, float]]]
```

Comment:
It's old and not correct any more, should be 
```suggestion
TTokenPred = t.Sequence[t.Sequence[t.Union[str, int]]]
```

Reply:
fixed